docs/accessibility-docs.md

```diff
@@ -0,0 +1,42 @@
+# Accessibility Docs
+
+If you are looking for accessibility docs[^1] for
```

Comment:
Maybe this is [the standard format for reference links](https://www.markdownguide.org/basic-syntax/#reference-style-links)?

```suggestion
If you are looking for[accessibility docs] [1] for
```